packages/usage-examples/scripts/code2md/index.js

```diff
@@ -0,0 +1,53 @@
+const parse = require('./parser');
+const { findTitle, generateHeader } = require('./helpers.js');
+
+/**
+ * Converts the code string to a markdown string
+ * @param {string} content
+ */
+function code2Md(content) {
+  const parsedArray = parse(content);
+  let output = parsedArray.join('\n');
+
+  if (output) {
+    const title = findTitle(output);
+    if (title) {
+      const header = generateHeader(title);
+      output = header + output;
+    }
+  }
+
+  return output;
+}
+
+module.exports = code2Md;
+
+/**
+ * If loaded from the command line.
+ *
+ * Options:
+ * --out: the output path (default to 'docs')
+ *
+ * All arguments will be parsed as files.
+ */
+const fromCLI = !module.parent;
+if (fromCLI) {
```

Comment:
CLI usage is a bit counter intuitive, I would have expected to be able to pass a glob (it actually works because ZSH, unsure how it would work without it)